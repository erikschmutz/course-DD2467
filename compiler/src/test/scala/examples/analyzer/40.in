type a = int;;
type b = a;;

(*====================
type a : int
type b : int
- :()
====================*)