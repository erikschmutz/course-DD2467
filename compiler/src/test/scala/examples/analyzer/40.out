AnalyzerResult(Unit(),Enviroment(List(TypeEntry(Identifier(a),Integer()),TypeEntry(Identifier(b),Integer()))))